trimesh/scene/cameras.py

```diff
@@ -307,6 +318,23 @@ def look_at(points, fov, rotation=None, distance=None, center=None):
     return cam_pose
 
 
+def _ray_pixel_coords(camera):
```

Comment:
What's the reason to break `_ray_pixel_coords` out from `camera_to_rays`? Is it used elsewhere?

Reply:
in `Camera.angles`

Reply:
Ah gotcha. Any reason to not make it "public-ish"?

Reply:
Happy to do so, I just don't think it's currently very well designed/named. I'm unsure if it'd be used much elsewhere, and whether people will know to find it even if it is. The description is something along the lines of

> the `xy` coordinates of pixels in the `z = -1` plane and the camera is looking along the `-z` axis. Reshaping to `res_x, res_y, 2` will give you the image coordinates except it's flipped up-down for some reason I don't understand.`

See comment below about flipping issue.